problem_builder/step_review.py

```diff
@@ -151,6 +157,14 @@ def student_view(self, context=None):
         html = loader.render_template("templates/html/sb-review-score.html", context.get("score_summary", {}))
         return Fragment(html)
 
+    def student_view_data(self, context=None):
+        context = context or {}
+
+        return {
+            'type': self.CATEGORY,
```

Comment:
@bradenmacdonald I was wrong to suggest to add `id: self.name` as only blocks deriving from the `QuestionMixin` support `self.name`. I added `id: self.name` to all blocks that support it.